shells/tests/arcs/ts/runtime/multiplexer-integration-test.ts

```diff
@@ -172,15 +161,15 @@ describe('Multiplexer', () => {
     };
     //
     const loader = new Loader(null, statics);
-    const context = await Manifest.parse(manifest, {fileName: './', loader, memoryProvider});
-    const runtime = new Runtime({loader, context, memoryProvider});
-    const arc = runtime.newArc('fooTest', storageKeyPrefixForTest());
+    const runtime = new Runtime({loader});
+    const context = await runtime.parseFile('./shells/tests/artifacts/polymorphic-muxing.recipes');
```

Comment:
this is not really `context` here, just a manifest. maybe rename the local var to avoid confusion?